soc/projects/numerics.md

```diff
@@ -82,6 +93,8 @@ and boundary cases.
 
 **Expected Results**: New and faster methods for evaluating properties of special functions.
 
+**Mentors:** [Alex Arslan](https://github.com/ararslan)
```

Comment:
Oh hey, that's me. What does being a mentor entail? Is there some guiding documentation I can consult?